lib/registry/src/lib.rs

```diff
@@ -1296,6 +1308,30 @@ pub struct BindingsGenerator {
     pub command: String,
 }
 
+impl Display for BindingsGenerator {
```

Comment:
Good catch! I didn't know that had been ported across from `wapm`, so I implemented it again.

Reply:
Actually, it's not quite the same as `SplitVersion` because this includes a command in the output and the struct has different fields. The `SplitVersion` type is also part of the `wasmer-cli` crate, so I can't import it.